src/main/java/creativei/dao/InquiryCustomDao.java

```diff
@@ -0,0 +1,21 @@
+package creativei.dao;
+
+import creativei.entity.City;
+import creativei.entity.Inquiry;
+import creativei.enums.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface InquiryCustomDao {
+    List<Inquiry> findByInquiryStatus(InquiryStatus status);
+
+    List<Inquiry> getFilteredInquiries(InquiryStatus status, CaseIndex caseIndex, Long cityId);
```

Comment:
Try using consistent method names. If other naming is findBy.. this should also be named similarly

Reply:
issue resolved in commit f372b98